### PR TITLE
fix(resolve-template): escape newline

### PR DESCRIPTION
**Pull request by glepur**
Properly handles new line characters in templates.

### CODE REVIEW

File: __tests__/resolve-template.test.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ test('replaces strings', async () => {
   expect(await resolveTemplate('123{{test}}456{{test}}{{test}}', { test: 'body' })).toBe('123body456bodybody')
   expect(await resolveTemplate('test{{test.foo}}test{{test.bar.baz}}test', { test: { foo: '111', bar: { baz: '222' } } })).toBe('test111test222test')
   expect(await resolveTemplate('{{ asd | default("te | st")}}', {})).toBe('te | st')
+  expect(await resolveTemplate('{{ asd | default("test\n") }}', {})).toBe('test\n')
```
Comment (ronag) [inline __tests__/resolve-template.test.js]: What about `test\n\n`?

**Comment by glepur** [comment]
Ah, global flag on .replace.
You got me :hand:

**Comment by ronag** [comment]
Please update the test as well...

**Comment by glepur** [comment]
? I did.

**Pull request merged**